api/v1alpha1/fenceagentsremediationtemplate_types.go

```diff
@@ -45,6 +45,8 @@ type FenceAgentsRemediationTemplateStatus struct {
 
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
+//+kubebuilder:resource:scope=Cluster,shortName=fart
```

Comment:
I know it's stupid, but that one makes me laugh every time, I think it's great :joy: